docs/architecture/microservices/secure-net-microservices-web-applications/azure-key-vault-protects-secrets.md

```diff
@@ -27,7 +27,7 @@ The **Microsoft.Extensions.Configuration.AzureKeyVault** package allows an ASP.N
    You can also use an overload of `AddAzureKeyVault` that takes a certificate in place of the client secret by just including a reference to the [Microsoft.IdentityModel.Clients.ActiveDirectory](https://www.nuget.org/packages/Microsoft.IdentityModel.Clients.ActiveDirectory) package.
 
 > [!IMPORTANT]
-> We recommend you to register Azure Key Vault as the last configuration provider, so it can override configuration values from previous providers.
+> It's a best practice to register Azure Key Vault as the last configuration provider, so it can override configuration values from previous providers.
```

Comment:
The original sentence was better

Reply:
I thought we shouldn't use "we"! when is it OK to do so?